configtx/msp.go

```diff
@@ -396,11 +402,10 @@ func parseCRL(crls [][]byte) ([]*pkix.CertificateList, error) {
 }
 
 func parsePrivateKeyFromBytes(priv []byte) (crypto.PrivateKey, error) {
-	if len(priv) == 0 {
-		return nil, nil
-	}
```

Comment:
@caod123 I removed this, so that cert, crl and private key handle nil in the same way. I'm not sure if this is appropriate, and whether the private key has some special considerations.

Reply:
The private key I believe is optional so should not error if nil

Reply:
https://github.com/hyperledger/fabric-config/blob/master/configtx/membership/membership.go#L23-L26

Reply:
I got it, thanks very much.